sormas-backend/doc/UserDataAccess.md

```diff
@@ -1,62 +1,131 @@
 
-# Case
+## Case
 * only users that are permitted to see cases at all
 * whoever created the case or is assigned to it is allowed to access it
-* access by region/district/community/health facility/point of entry of the case
+* all cases accessible for National users
+* access by jurisdiction, that is based region/district/community/health facility/point of entry of the case
 * cases that are made public can be accessed by all users permitted to see cases (this feature can be disabled)
+* edit is possible only for cases that are part of the users jurisdiction
+* view of personal and sensitive data is possible only for cases that are part of the user's jurisdiction
 
-# Contact
+## Contact
 * only users that are permitted to see contacts at all
 * whoever created it or is assigned to it is allowed to access it
+* all contacts accessible for National users
 * users see all contacts of their cases
-* access by region/district of the contact
+* access by jurisdiction, that is using region/district of the contact and jurisdiction of the case the contact is assigned to
+* edit is possible only for contacts that are part of the users jurisdiction
+* view of personal and sensitive data is possible only for contacts that are part of the user's jurisdiction
 
-# Visits
+## Visits
 * users see all visits of the user's contacts
+* view of sensitive data is possible only for contacts that are part of the user's jurisdiction
 
-# Event
+## Event
 * only users that are permitted to see events at all
 * whoever created the event or is assigned to it is allowed to access it
+* all events accessible for National users
 * access by region/district of the event
 
-# EventParticipant
+## EventParticipant
 * users see all participants of all events they can access
+* view of personal and sensitive data is possible only for events that are part of the user's jurisdiction
 
-# Tasks
+## Tasks
 * only users that are permitted to see tasks at all
 * whoever created the task or is assigned to it is allowed to access it
-* all tasks for the user's cases
-* all tasks for the user's contacts
-* all tasks for the user's events
-
-# Samples
+* all tasks accessible for National users
+* all tasks for the user's cases except the ones assigned to users in other jurisdiction
+* all tasks for the user's contacts except the ones assigned to users in other jurisdiction
+* all tasks for the user's events except the ones assigned to users in other jurisdiction
+ 
+## Samples
 * only users that are permitted to see samples at all
+* all samples accessible for National Users
 * whoever created the sample or is assigned to it is allowed to access it
 * users see all samples of their cases
 * lab users see all samples of their laboratory
+* access by jurisdiction of the case or contact the sample belongs to
+* edit is possible only for samples of cases and/or contacts in the user's jurisdiction
```

Comment:
Samples can now also be added for event participants, so this should be reflected here